components/pkg-export-kubernetes/src/bind.rs

```diff
@@ -27,6 +28,32 @@ pub struct Bind {
 impl FromStr for Bind {
     type Err = Error;
 
+    /// Create a `Bind` from a string.
+    ///
+    /// # Errors
+    ///
+    /// * `bind_spec` is not in the form `"name:service:group"`
```

Comment:
If this is a Habitat bind, shouldn't it be `name:service.group`?

Also, if it is a Habitat bind, you might want to use the type from the Supervisor (like you do for `Topology`... there are other parts of a Bind that might get used which would break this.

(which might lend more weight to the "pull the common types up to a shared crate" argument below)

Reply:
Yeah we've had this typo for a bit now even though you pointed out in a previous review. Fortunately converting the code to make use of use `habitat_sup::manager::service::ServiceBind` will automatically solve the typo.